packages/liferay-js-insights/README.md

```diff
@@ -0,0 +1,69 @@
+# liferay-js-insights
+
+> A simple dependency analysis reporter. It scans ES6 modules for imports and generate a report on used external packages.
+
+## Installation
+
+Either:
+
+-   Install globally: `yarn global add liferay-js-insights`; or:
+-   Use the latest without installing: `npx liferay-js-insights` (see below for usage).
+
+## Usage
+
+```
+liferay-js-insights [option...]
+
+Options:
+  --airtable    [optional: dumps the analysis to a remote airtable installation]
+  --apiKey      [optional: airtable API KEY]
+  --baseKey     [optional: airtable base]
+  --help
+  --json        [optional: dumps the analysis as a json file]
+```
+
+### Default Report
+
+By default, a table-based report will be displayed on screen unless a different reporter is selected.
+
+`npx liferay-js-insights src/**/*.es.js`
+
+![Default Table Report](/docs/img/report_table.png)
+
+### JSON Report
+
+When the `--json` flag is passed, collected insights will be dumped in a `${name}.json` file. The name of the file can be passed using the `--name` flag. If no flag is passed the name will be that of the cwd last folder.
```

Comment:
Maybe when no filename is given, printing to standard-out would be a good behavior (or failing that, I should be able to make it print to standard out by specifying "-" as a filename, which is a common convention in command-line tools).

Reply:
So, you would also expect to be able to see the json report in a terminal? We'd need to pretty-print it then :)

Reply:
TBH, I wasn't thinking about eyeballing it but rather piping it into another tool. Making your command-line tools pipe-friendly is a good practice.

But yeah, if you want to make it pretty, a `JSON.stringify(..., null, 2)` wouldn't hurt.

Reply:
Makes sense, gotta love piping!